packages/react-instantsearch-core/src/core/__tests__/createConnector.js

```diff
@@ -417,7 +417,7 @@ describe('createConnector', () => {
       });
 
       expect(onSearchParameters.mock.calls).toHaveLength(1);
-      expect(onSearchParameters.mock.calls[0][0]).toBe(getSearchParameters);
+      expect(onSearchParameters.mock.calls[0][0]).toMatchSnapshot();
```

Comment:
We can remove the snapshot and use a matcher to expect the argument to be a function.

```js
expect(onSearchParameters.mock.calls[0][0]).toEqual(expect.any(Function));
```